improver_tests/nbhood/square_kernel/test_SquareNeighbourhood.py

```diff
@@ -478,83 +395,75 @@ def test_multiple_times(self):
                 [1.0, 1.0, 1.0, 1.0, 1.0],
             ]
         )
-        cube = set_up_cube(
-            zero_point_indices=((0, 0, 2, 2), (0, 1, 1, 2)),
-            num_time_points=2,
-            num_grid_points=5,
-        )
+
+        cube = self.multi_cube
```

Comment:
This cube doesn't contain multiple times - it contains multiple realizations.  In this case I would suggest changing the name of the test and the docstring rather than creating a different input cube.  This is because the lead time dependent functionality is tested elsewhere (in the `test_BaseNeighbourhoodProcessing` module).

Reply:
Done.